libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/LedgerTypes/Specs.hs

```diff
@@ -291,16 +304,15 @@ certStateSpec ::
   Term fn AccountState ->
   Term fn EpochNo ->
   Specification fn (CertState era)
-certStateSpec ctxDelegatees acct epoch = constrained $ \ [var|certState|] ->
+certStateSpec drepRoleCredSet acct epoch = constrained $ \ [var|certState|] ->
   match certState $ \ [var|vState|] [var|pState|] [var|dState|] ->
     [ satisfies pState (pstateSpec epoch)
     , reify pState psStakePoolParams $ \ [var|poolreg|] ->
         [ dependsOn dState poolreg
-        , satisfies dState (dstateSpec acct poolreg)
+        , satisfies dState (dstateSpec drepRoleCredSet acct poolreg)
         ]
     , reify dState getDelegatees $ \ [var|delegatees|] ->
         [ satisfies vState (vstateSpec epoch delegatees)
```

Comment:
the getDelegatees, computes the inverse relation for the VState from the DState, and the vStateSpec, ensures this inverse holds.